resources/run_adf.sh

```diff
@@ -333,7 +335,7 @@ if [[ $? -ne 0 ]]; then
     mkdir -p $TMP_DIR
     mkdir -p $LOGS_PATH
     # Start mongohoused with appropriate config
-    nohup $GO run -tags mongosql ./cmd/mongohoused/mongohoused.go \
+    $GO run -tags mongosql ./cmd/mongohoused/mongohoused.go \
```

Comment:
The nohup here was causing an issue with mac on evergreen to not start adf.  It shouldn't affect evergreen since things are run in one session.  For users running adf locally, the terminal that runs this script should not be closed.

Reply:
Seems fine to me. I've had issues with nohup on macos, too.